### ISSUE TITLE
Make using prefix configurable

### ISSUE THREAD

**Issue by SimonMonecke** [flosell/lambdacd-cctray#9]
Hi Flosell,

we don't need the new prefix feature because we have separate cctrays for our pipelines.
It would be nice if we could disable the use of prefixes. 

```
use-prefix (... config)
name (if (and use-prefix pipeline-name)
    (str pipeline-name " :: " step-name)
    step-name)
```

Regards,
  Miriam & Simon

**Comment by flosell** [comment]
Thanks for the feedback, sounds reasonable. Since you already have a clear picture of the changes necessary (add a test and you are good), do you want to take a swing creating a PR for this? 

If you do, the contribution guidelines are similar to those of LambdaCD, i.e. update tests, changelog and readme as appropriate.

**Comment by SimonMonecke** [comment]
We did not send you a PR because we do not know how to configure LambdaCD-CCtray.
We could add a optional parameter to provide a config-map:

```
(defn cctray-handler-for
  ([pipeline]
   (cctray-handler-for pipeline {:fallback-base-url nil}))
  ([pipeline {fallback-base-url :fallback-base-url :as config}]
```

If this is okay for you we will send you a PR.

Regards,
  Simon

**Comment by flosell** [comment]
I would just use the config map of the LambdaCD pipeline, like we already
do for the `:ui-url` config. Makes it easier to add config parameters
without a breaking api change and to work with multiple pipelines.

**Comment by flosell** [comment]
Resolved by PR #10
